src/panel/details.tsx

```diff
@@ -100,7 +100,29 @@ interface DetailsProps { result: Result, height: IObservableValue<number> }
                                                                 }}>
                                                                 {result._log._uri.file}{result._log._uriUpgraded && ' (upgraded)'}
                                                             </a>
-                            {/* <span>Properties</span>		<span><pre><code>{JSON.stringify(selected.properties, null, '  ')}</code></pre></span> */}
+                            {(() => {
+                                // Rendering "tags" reserved for a future release.
+                                const { tags, ...rest } = result.properties ?? {};
+                                return <>
+                                    <span>&nbsp;</span><span></span>{/* Blank separator line */}
+                                    {Object.entries(rest).map(([key, value]) => {
+                                        return <Fragment key={key}>
+                                            <span className="ellipsis">{key}</span>
+                                            <span>{(() => {
+                                                if (value === null)
+                                                    return '—';
+                                                if (Array.isArray(value))
+                                                    return <span style={{ whiteSpace: 'pre' }}>{value.join('\n')}</span>;
+                                                if (typeof value === 'boolean')
+                                                    return JSON.stringify(value, null, 2);
```

Comment:
do we need to use `null, 2`? In this case, isn't this similar to JSON.stringify(value)? The same comment applies for the next stringify.

Reply:
or...wait...looking at the docs, it will add two spaces at the end. If that's correct, is that really needed?

Reply:
The intent was to have pretty-printed JSON and the `null, 2` was for that.

Reply:
"2" means "indent with 2 spaces". It is not adding spaces at the end.